compiler/rustc_parse/src/parser/attr.rs

```diff
@@ -8,16 +8,18 @@ use rustc_span::{sym, Span};
 
 use tracing::debug;
 
+// Public for rustfmt usage
```

Comment:
You cannot make this private if `parse_attribute` is public, so the comment seems unnecessary.

Reply:
I can remove this if you'd prefer, but I've been trying to be very explicit about calling out instances where rustfmt ends up needing a higher visibility on something within the compiler than what would've been done otherwise.

Naturally the public visibility on the function requires this to be public as well, and in this case the two happen to be in the same file and fairly close together. However, I just think that for anyone looking at this code down the road and wondering "why is this enum public?" that it's beneficial to have the inline comment in context vs. having to do a search to find the comment on the function or reducing the visibility to try to let the compiler tell them.

Let me know how you'd like to proceed

Reply:
Ok, leaving this at your discretion.